cypress/e2e/shared/explorer.test.ts

```diff
@@ -1198,18 +1198,12 @@ describe('DataExplorer', () => {
         cy.getByTestID('save-as-dashboard-cell--submit').click()
 
         // wait some time for save
-        cy.wait(200)
-        // ensure dashboard created with cell
-        cy.get('@org').then(({id: orgID}: Organization) => {
-          cy.fixture('routes').then(({orgs}) => {
-            cy.visit(`${orgs}/${orgID}/dashboards/`)
-            cy.getByTestID('dashboard-card--name')
-              .contains(dashboardCreateName)
-              .should('exist')
-              .click()
-            cy.getByTestID(`cell ${cellName}`).should('exist')
-          })
-        })
+        cy.wait(1000)
```

Comment:
sniff for the url change (cy.location or something) instead of having an arbitrary wait time